src/views/ContinuationOut.vue

```diff
@@ -712,13 +759,13 @@ export default class ContinuationOut extends Mixins(CommonMixin, DateMixin,
 
     const data: any = {
       [FilingTypes.CONTINUATION_OUT]: {
-        // FUTURE: add more filing properties below and change the dummy data to actual data
         details: `${this.defaultComment}\n${this.detailComment}`,
-        continuationOutDate: '2023-08-23T23:53:32.000+00:00',
+        continuationOutDate: this.effectiveDate,
         foreignJurisdiction: {
-          country: 'Canada',
-          legalName: '0871322 B.C. LTD.'
-        }
+          country: this.selectedCountry,
+          region: this.selectedRegion
+        },
+        legalName: this.businessName
       }
```

Comment:
Should the court order object be saved here (and presumably, loaded above)?

Reply:
It's being saved on line 773!